src/canister/individual_user_template/Cargo.toml

```
@@ -11,12 +11,14 @@ crate-type = ["cdylib"]
 [dependencies]
 candid = { workspace = true }
 ciborium = { workspace = true }
-ic-stable-structures = {workspace = true }
+ic-stable-structures = "0.6.2"
```

Workspace has also has the upgraded version now

```suggestion
ic-stable-structures = { workspace = true }
```